vue/sbc-common-components/src/services/token.services.ts

```diff
@@ -38,7 +38,7 @@ class TokenServices {
     })
   }
 
-  scheduleRefreshTimer (refreshEarlyTime :number = 0) {
+  scheduleRefreshTimer (refreshEarlyTime: number = 0) {
```

Comment:
did u intend to commit this as well?  the token changes

Reply:
Yes, cleanup.

Reply:
ok..

Reply:
I made these changes [not the linting one] in the other PR..so might be a conflict

Reply:
I reverted these changes.